style.css

```diff
@@ -1,3 +1,399 @@
+  }
+
+  .hamburger-menu-display {
+    display: none;
+  }
+
+  .header-menu-mobile {
+    display: none;
+  }
+
+  .header-menu-mobile.open {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 500px) {
+  .header-seconday-mobile {
+    display: none;
+}
+}
+
+.header-secondary {
+  display: none;
+}
+
+@media only screen and (min-width: 500px) {
+  .header-secondary {
+    background-color: #cba6a6;
+    background-color: #ca2e2e;
+    background-color: #0862d7;
+    display: grid;
+    grid-template-columns: repeat(10, 1fr);
+    /* column-gap: 10px; */
+    font-size: 1.1em;
+    padding: 10px 0 10px 20px;
+    margin: 0;
+    /* border: 2px solid red; */
+  }
+
+  .nav-list {
+    list-style: none;
+  }
+
+  .nav-anchor {
+    text-decoration: none;
+    color: white;
+    padding: 10px;
+  }
+
+  .nav-anchor:hover {
+    background-color: #403d3d;
+    transition: all 1s ease;
+    padding: 10px;
+  }
+}
+
+.form {
+  display: block;
+  background-color: #eee9e9;
+  border-radius: 2px;
+  padding: 15px;
+  margin: 20px 20px 0 20px;
+  /* border: 2px dotted red; */
+}
+
+.search-results {
+  color: black;
+  font-size: 1.3em;
+  padding-left: 30px;
+}
+
+@media only screen and (min-width: 500px) {
+  .form {
+    display: block;
+    background-color: #eee9e9;
+    border-radius: 2px;
+    padding: 10px 0 10px 20px;
+    margin: 20px 20px 0 20px;
+    /* border: 2px dotted red; */
+  }
+}
+
+#main-title {
+  /* text-decoration: underline; */
+  
+  word-spacing: .2em;
+  /* border-bottom: 5px solid #d4cfcf; */
+ 
+  font-size: 25px;
+  text-align: center;
+  padding: 40px 10px 10px 15px;
+  margin: 0;
+}
+
+.flag-img {
+  width: 25px;
+  height: auto;
+  padding: 0;
+  margin: 0 2px -5px 2px;
+  /* border: 2px solid red; */
+}
+
+.form {
+ display: grid;
+ grid-template-rows: 1fr;
+}
+
+#select-field {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.search-input {
+ 
+}
+
+@media only screen and (min-width: 500px) {
+  .form {
+    display: block;
+  }
+
+  #select-field {
+width: 30%;
+  }
+  
+  #search-input {
+  width: 30%;
+  }
+  
+}
+
+
+
+
+@media only screen and (min-width: 500px) {
+    h1 {
+    /* border-top: 15px solid #917b7b; */
+    /* border-bottom: 8px solid #917b7b; */
+    border-bottom: none;
+    word-spacing: .5em;
+    color: black;
+    /* position: fixed; 
+    top: 0;
+    left: 0; */
+    padding: 50px 0 0 0;
+    width: 100%;
+    margin-top: 0;
+    /* border: 2px dotted red; */
+  }
+
+  .flag-img {
+    width: 25px;
+    height: auto;
+    padding: 0;
+    margin: 0 2px -5px 2px;
+    /* border: 2px solid red; */
+  }
+
+  .channel-span {
+    font-size: .9em;
+    font-weight: 400;
+  }
+}
+
 #root {
   color: red;
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 40px;
+  padding: 20px 25px 50px 25px;
+}
+
+@media only screen and (min-width: 500px) {
+  #root {
+    color: red;
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Comment:
Take a look at performing a search that returns 1 item and try changing the above line of code from
`grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));` 
to
`grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));`
This will prevent the single item from filling the screen.

Reply:
Hi Pete, can you explain this a bit more please?